cts/query/jsonSchema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "double": {
+    "request": {
+      "find": "values",
+      "filter": {
+        "$jsonSchema": {
+          "bsonType": "object",
```

Comment:
Have tried to use `required` field here, but it's not supported by us. Should I create an issue?

Reply:
Yes, in the FerretDB repo